scss/components/breadcrumb.scss

```diff
@@ -67,3 +67,26 @@ $block: #{$fd-namespace}-breadcrumb;
         }
     }
 }
+
+a {
+    text-decoration: none;
+    display: inline-block;
+    transition: all $fd-animation--speed ease-in;
+    @include fd-var-color("color", $fd-color--action, --fd-color-action);
+    @include fd-hover() {
+      @include fd-var-color("color", fd-color-state("hover", "action"), --fd-color-action-hover);
+    }
+    @include fd-active-pressed-selected() {
+      @include fd-var-color("color", fd-color-state("selected", "action"), --fd-color-action-selected);
+      outline: none;
+    }
+    &:focus {
+      outline-style: dotted;
+      outline-width: 1px;
+      @include fd-var-color("outline-color", fd-color-state("hover", "action"), --fd-color-action-focus);
+    }
+    @include fd-disabled() {
+      @include fd-var-color("outline-color", fd-color-state("disabled", "action"), --fd-color-action-disabled);
+      cursor: not-allowed;
+    }
+}
```

Comment:
Styles should not be done on plain elements (`<a>` in this case) otherwise these styles can bleed into other components or elements on a page.

Reply:
I think that block could be switched over to be the `.fd-breadcrumb__link` class.